app/src/main/java/ar/com/wolox/android/example/ui/login/LoginPresenter.java

```diff
@@ -29,8 +43,14 @@
      */
     public void onLoginButtonClicked(String email, String pass, Context context) {
 
+        if (mUsers == null) {
+            getUsersFromServer();
```

Comment:
Since `getUsersFromServer()` is an asynchronous method, it should not be used normally as if it were synchronous.